yt_dlp/extractor/ted.py

```diff
@@ -300,68 +233,25 @@ def _talk_info(self, url, video_name):
 
-        video_id = compat_str(talk_info['id'])
+        # trim thumbnail resize parameters
+        thumbnail = self._search_regex(r'^(http[^?]*)', playerData.get('thumb'), 'thumbnail', default=None) or self._search_regex(r'^(http[^?]*)', self._og_search_property('image', webpage), 'thumbnail', default=None)
 
-        return {
-            'id': video_id,
-            'title': title,
-            'uploader': player_talk.get('speaker') or talk_info.get('speaker'),
-            'thumbnail': player_talk.get('thumb') or talk_info.get('thumb'),
-            'description': self._og_search_description(webpage),
-            'subtitles': self._get_subtitles(video_id, talk_info),
-            'formats': formats,
-            'duration': float_or_none(talk_info.get('duration')),
-            'view_count': int_or_none(data.get('viewed_count')),
-            'comment_count': int_or_none(
-                try_get(data, lambda x: x['comments']['count'])),
-            'tags': try_get(talk_info, lambda x: x['tags'], list),
-        }
-
-    def _get_subtitles(self, video_id, talk_info):
-        sub_lang_list = {}
-        for language in try_get(
-                talk_info,
-                (lambda x: x['downloads']['languages'],
-                 lambda x: x['languages']), list):
-            lang_code = language.get('languageCode') or language.get('ianaCode')
-            if not lang_code:
-                continue
-            sub_lang_list[lang_code] = [
-                {
-                    'url': 'http://www.ted.com/talks/subtitles/id/%s/lang/%s/format/%s' % (video_id, lang_code, ext),
-                    'ext': ext,
-                }
-                for ext in ['ted', 'srt']
-            ]
-        return sub_lang_list
-
-    def _watch_info(self, url, name):
-        webpage = self._download_webpage(url, name)
-
-        config_json = self._html_search_regex(
-            r'"pages\.jwplayer"\s*,\s*({.+?})\s*\)\s*</script>',
-            webpage, 'config', default=None)
-        if not config_json:
-            embed_url = self._search_regex(
-                r"<iframe[^>]+class='pages-video-embed__video__object'[^>]+src='([^']+)'", webpage, 'embed url')
-            return self.url_result(self._proto_relative_url(embed_url))
-        config = json.loads(config_json)['config']
-        video_url = config['video']['url']
-        thumbnail = config.get('image', {}).get('url')
-
-        title = self._html_search_regex(
-            r"(?s)<h1(?:\s+class='[^']+')?>(.+?)</h1>", webpage, 'title')
-        description = self._html_search_regex(
-            [
-                r'(?s)<h4 class="[^"]+" id="h3--about-this-talk">.*?</h4>(.*?)</div>',
-                r'(?s)<p><strong>About this talk:</strong>\s+(.*?)</p>',
-            ],
-            webpage, 'description', fatal=False)
+        # tags
+        tags = try_get(playerData, lambda x: x['targeting']['tag'], str)
+        if tags:
+            tags = tags.split(',')
```

Comment:
```suggestion
        tags = try_get(playerData, lambda x: x['targeting']['tag'].split(','))
```
can be inlined in return